src/templates/app/user/authenticate.jsx

```diff
@@ -36,6 +36,11 @@ const Authenticate = () => (
             <h1 className='gr-padding-10'>{it.L('Your account has been verified successfully')}</h1>
         </div>
 
+        <div id='authentication_unneeded' className='center-text gr-padding-20 invisible'>
+            <img className='gr-padding-20' src={it.url_for('images/pages/authenticate/invalid.svg')} />
+            <h1 className='gr-padding-10'>{it.L('You do not need to authenticate your account at this time.[_1]We will inform you when your account needs to be authenticated.', '<br />')}</h1>
```

Comment:
can you use the <br /> instead of a placeholder and add a separate localize tag for the second sentence?